activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+*   Adds `Arel::FactoryMethods#cast(node, type)`
+
+    For example:
+
+    ```ruby
+      product_table = Product.arel_table
+      product_table.cast(product_table[:position], "integer")
+    ```
+
+    Produces SQL:
+
+    ```sql
+      CAST(position as integer)
+    ```
+
+    *Dr Nic Williams*
+
```

Comment:
Arel isn't part of the public API of the framework, so changes to it should not be documented to the CHANGELOG that only list changes to the public API of the framework
```suggestion
```

Reply:
Ok removed.